ex1/ex1Spec.js

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Did you not get the require working (or maybe you just did it for another question) and instead you have to copy the function?